proxy/http/HttpTransactHeaders.cc

```diff
@@ -1073,3 +1073,60 @@ HttpTransactHeaders::remove_privacy_headers_from_request(HttpConfigParams *http_
     }
   }
 }
+
+void
+HttpTransactHeaders::normalize_accept_encoding(const OverridableHttpConfigParams *ohcp, HTTPHdr *header)
+{
+  int normalize_ae = ohcp->normalize_ae;
+
+  if (normalize_ae == 99) { // Not explicitly configured.
```

Comment:
What is 99? Where did that come from? 1 should be the default, no?

Reply:
99 is the default value in RecordsConfig.cc, for both normalize_ae_gzip and normalize_ae.  It is the solution I used to make sure that explicitly configured values would always override implicit defaults.

Reply:
Ok, that's not cool. Either "0" or "-1" is what we normally use, 99 seems much to arbitrary IMO.

Reply:
OK, that should be fine, I am probably just paranoid to worry that int8_t won't sign extend properly.